pkg/gatherers/clusterconfig/service_accounts_test.go

```
@@ -52,32 +82,23 @@ func Test_ServiceAccounts_Gather(t *testing.T) {
 			t.Parallel()
 			coreClient := kubefake.NewSimpleClientset()
 			for _, d := range test.data {
-				_, err := coreClient.CoreV1().Namespaces().Create(
-					context.Background(),
-					&corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: d.Namespace}}, metav1.CreateOptions{},
-				)
-				if err != nil {
-					t.Fatalf("unable to create fake ns %s", err)
+				namespace, _ := coreClient.CoreV1().Namespaces().Get(context.Background(), d.Namespace, metav1.GetOptions{})
```

Do not ignore the error please